docs/src/Datapoint.doc.js

```diff
@@ -0,0 +1,140 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Datapoint } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+import FeedbackCallout from './components/FeedbackCallout.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(<FeedbackCallout componentName="Datapoint" />);
+
+card(
+  <PageHeader
+    name="Datapoint"
+    description="Datapoint displays at-a-glance data for a user to quickly view key metrics."
+    defaultCode={`
+<Datapoint size="lg" infoText="Contextual information for this Datapoint" title="Datapoint title" value="1.23M" trend={{value: 30, accessibilityLabel: "Trending up"}} helperText="Datapoint helper text" />
+`}
+    pilot
+  />,
+);
+
+// Specifying Component gives auto warnings about missing props
+card(
+  <PropTable
+    Component={Datapoint}
+    props={[
+      {
+        name: 'title',
+        type: 'string',
+        description: 'The header for the component',
```

Comment:
These should include links to the relevant sections below. Check out Tooltip for good examples for this

Reply:
Done!